_gsocproposals/2018/proposal_ROOTmpi.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Message Passing Interface for ROOT (ROOTMpi)
+layout: gsoc_proposal
+project: ROOT
+year: 2018
+organization:
+  - OProject
+  - CERN
+  - UdeA
+---
+
+# Description
+
+By standardizing the way different machines communicate during a running process, we can analyze bigger chunks of data in less time. ROOT MPI allows to communicate ROOT’s native objects on top of the C/C++ raw data types. ROOT’s serialization methods and optimal design of the new C++ standard permits the user to focus on the the algorithm instead of low level syntax.
```

Comment:
"allows to communicate" is pretty bad English. Suggest instead, "ROOT MPI allows communication of ROOT’s native objects..."

Reply:
"on the the algorithm" - remove duplicate "the"